qsa-api/qsa_api/app.py

```diff
@@ -15,6 +15,7 @@
 
 class QSA:
     def __init__(self) -> None:
+        print("FFFFF")
```

Comment:
This commit piqued my curiosity ^^

Reply:
Sorry, I thought I was on a different branch😅 
I've deleted it.